### PR TITLE
Learn - Archived cards not appearing and ShuffleDeck working

### PR DESCRIPTION
**Pull request by MwaniN**
Archived cards no longer appear in the study mode and shuffling the deck now works

### CODE REVIEW

**Comment by MwaniN** [comment]
[[Mwani] Implementation 6.2.2 : Study page shuffle button](https://trello.com/c/zPR7ACAe/70-mwani-implementation-622-study-page-shuffle-button)

File: src/App.tsx
Comment (stephencapper) [inline src/App.tsx:1]: Can we keep the "test this" component out of App in main for now?

Comment (MwaniN) [reply src/App.tsx:1]: @stephencapper For sure! Had that just to see the functionality as I worked on it, will leave it out of future pushes if I continue to use it before making the final view

Comment (stephencapper) [reply src/App.tsx:1]: Thank you!

**Review by stephencapper — APPROVED** [review]
Looks good - thank you!

**Pull request merged**